src/components/Header/Tools.tsx

```
@@ -324,7 +324,7 @@ const Tools = () => {
       });
     };
 
-    const AskRedHatIcon = () => <img src="/apps/frontend-assets/technology-icons/ai-chat-ask-redhat.svg" alt="Ask Red Hat" className="chr-c-ask-redhat-icon" />;
+    const AIExperienceIcon = () => <img src="/apps/frontend-assets/technology-icons/rh-ui-icon-ai-experience.svg" alt="AI Experience" className="chr-c-ai-experience-icon" />;
```

ESLint

[error] Replace `<img·src="/apps/frontend-assets/technology-icons/rh-ui-icon-ai-experience.svg"·alt="AI·Experience"·className="chr-c-ai-experience-icon"·/>` with `(⏎······<img·src="/apps/frontend-assets/technology-icons/rh-ui-icon-ai-experience.svg"·alt="AI·Experience"·className="chr-c-ai-experience-icon"·/>⏎····)`

(prettier/prettier)